src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -38,7 +50,7 @@
     <None Include="project.json" />
     <None Include="windows\project.json" />
```

Comment:
@mconnew has the details but these are dependencies needed for a specific WCF scenario to work. These dependencies are not supported on Linux which is why they are pulled in on a Windows only condition.